main_routers/characters_router.py

```
@@ -0,0 +1,1127 @@
+            }, status_code=500)
+            
+        return {"success": True, "message": "音色注册成功"}
+    except Exception as e:
+        return JSONResponse({
+            'success': False,
+            'error': str(e)
+        }, status_code=500)
+
+
+@router.post('/voice_clone')
+async def voice_clone(file: UploadFile = File(...), prefix: str = Form(...)):
```

Ruff (0.14.8)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)